sdk/monitor/monitor-ingestion/src/gZippingPolicy.browser.ts

```diff
@@ -11,7 +11,10 @@ export const gZippingPolicyName = "GzippingPolicyForBrowser";
 export const GZippingPolicy: PipelinePolicy = {
   name: gZippingPolicyName,
   sendRequest: async (req, next) => {
-    // This is a no-op for now, will be implementing gzipping for browser using pako https://github.com/Azure/azure-sdk-for-js/issues/22593
+    if (req.body) {
+      const buffer = pako.gzip(req.body.toString());
```

Comment:
it really is this easy? Wow!

I don't know why, but I never want to trust `toString` and end up stringifying this way: 

```suggestion
      const buffer = pako.gzip(String(req.body));
```

Maybe your way is better if we know `body` is always a non-null object? I'm not sure